.github/workflows/ci.yml

```
@@ -706,4 +706,37 @@ jobs:
+          fail_ci_if_error: true
+
+  test-redis-connection-docker:
+    name: '[Test] Redis Docker Connection'
+    needs: build-redis-latest-manual
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)